assets/prometheus-k8s/service-account.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: v1
+automountServiceAccountToken: true
```

Comment:
Although all other tests except aws single node was passing without this. I was still unable to bring cluster using cluster bot. Upon checking must-gather this needs service account access too https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp-modern/1496318026973188096

Reply:
I was able to bring up [cluster](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp-modern/1496362149478404096) after this change. Enabled workload monitoring from cluster since that prometheus needed service account too enabled it that as well 38ad99f723f177fcc47c1de48fab6114b3bc3128